pkg/stats/tracee_stats.go

```diff
@@ -0,0 +1,9 @@
+package stats
+
+type TraceeStats struct {
```

Comment:
I suggest to put TraceeStats and EngineStats in the same file, and rename the file to stats.go under package stats.
Also, maybe rename to something more meaningful to tracee-ebpf (as these counters are relevant to ebpf side)

Reply:
Renamed to EbpfStats